build.fsx

```diff
@@ -329,34 +329,32 @@ Target "BuildPackage" DoNothing
 
 
 // --------------------------------------------------------------------------------------
-// .NET CLI and .NET Core
+// .NET Core SDK and .NET Core
 
 let assertExitCodeZero x = if x = 0 then () else failwithf "Command failed with exit code %i" x
-let netcoreFW = "netstandard1.5"
 
-Target "DotnetCliBuild" (fun _ ->
+Target "Build.NetCore" (fun _ ->
     Shell.Exec("dotnet", "restore") |> assertExitCodeZero
-    Shell.Exec("dotnet", sprintf "--verbose build --framework %s --configuration Release" netcoreFW, "src/Chessie") |> assertExitCodeZero
+    Shell.Exec("dotnet", "--verbose pack --configuration Release", "src/Chessie") |> assertExitCodeZero
```

Comment:
yes, but pack is at https://github.com/fsprojects/Chessie/pull/46/files/589dae8b4198e41aad71ccc688d2834c4eab1c20#diff-6074983490330a0aac1bd23c958232d7R352